sdk/storage/storage-blob-changefeed/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "12.0.0-preview.3",
   "description": "Microsoft Azure Storage SDK for JavaScript - Blob Change Feed",
   "main": "./dist/index.js",
-  "module": "./dist-esm/storage-blob-changefeed/src/index.js",
+  "module": "dist-esm/src/index.js",
```

Comment:
The old path could be intentional due to shared common code? @ljian3377 @sadasant

Reply:
Reverted in https://github.com/Azure/azure-sdk-for-js/pull/13710/commits/aa7ff460f8190dc78779a5ded8945de71225d141